design-assets/README.md

```diff
@@ -0,0 +1,14 @@
+You will find the following in the `design-assets` folder:
+
+- CMSDS Sketch file
+- CMSDS fonts (Bitter,Open Sans, and Font Awesome)
+
+### Using the CMSDS Sketch Library
+
+We encourge usage of the CMSDS Sketch Library. When you update components in a Library, documents that use them will get an update notification. You can review those changes and confirm them to make sure you’re always using the most up-to-date components in your work.
+
+- Visit the design.cms.gov homepage and click the `get the Sketch Library` link.
+  OR
+- paste this url into your browser `sketch://add-library?url=https%3A%2F%2Fgithub.com%2FCMSgov%2Fdesign-system%2Fraw%2Fmaster%2Fdesign-assets%2Fcms-design-system-ui-kit.xml`
+
+Sketch will then open and download the CMSDS Sketch Library.
```

Comment:
It might be good to either explain how to use symbols and styles from the Sketch library in your files (and that you ideally shouldn't be working directly in the Sketch library or duplicating it for a new project), or else link to Sketch's documentation on it. I'm assuming we'll have people who are unfamiliar with how the Libraries feature works.

Reply:
Maybe screenshots to show the kit being successfully added to the library, then showing an example of adding a symbol from the library into a new sketch file

![image](https://user-images.githubusercontent.com/4106356/79365984-43a78100-7f00-11ea-978e-438bf8e95d6b.png)

![image](https://user-images.githubusercontent.com/4106356/79366069-633ea980-7f00-11ea-90e9-10e1cc26f315.png)